.github/workflows/daily_ete_test.yml

```diff
@@ -23,16 +23,80 @@ on:
         description: 'Set testcase module filter: chat, restful, pipeline, quantization. Default contains all models'
         type: string
         default: "['quantization','convert','pipeline','restful','chat','interface-pipeline']"
+      tools_regression:
+        required: true
+        description: 'Whether start a tool regression'
+        type: boolean
+        default: true
+      restful_regression:
+        required: true
+        description: 'Whether start a restful api regression'
+        type: boolean
+        default: false
+      triton_regression:
+        required: true
+        description: 'Whether start a triton server api regression'
+        type: boolean
+        default: true
   schedule:
     - cron:  '00 21 * * *'
 
 env:
   HOST_PIP_CACHE_DIR: /nvme/github-actions/pip-cache
   HOST_LOCALTIME: /usr/share/zoneinfo/Asia/Shanghai
+  OUTPUT_FOLDER: cuda11.8_dist_${{ github.run_id }}
 
 
 jobs:
-  test_functions:
+  linux-build:
+    strategy:
+      matrix:
+        pyver: [py38, py310]
+    runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: ${{ matrix.pyver }}
+      PLAT_NAME: manylinux2014_x86_64
+      DOCKER_TAG: cuda11.8
+    steps:
+      - name: Free disk space
+        uses: jlumbroso/free-disk-space@main
+        with:
+          # This might remove tools that are actually needed, if set to "true" but frees about 6 GB
+          tool-cache: false
+          docker-images: false
+          # All of these default to true, but feel free to set to "false" if necessary for your workflow
+          android: true
+          dotnet: true
```

Comment:
what are dotnet and haskell?

Reply:
https://github.com/jlumbroso/free-disk-space. According to readme and running log. I think the values should be true. But I think it maybe unnecessary in this workflow. I will try it.